client/src/components/guests.js

```diff
@@ -5,34 +5,95 @@ import './Guests.css'
 
 class Guests extends React.Component {
   state= {
-    card: false
+    card: false,
+    adults: 1,
+    children: 0,
+    infants: 0,
   }
 
-  showCard = () => {
+
+updateParentState = () => {
+  const guests = {}
+  
+  guests.adults = this.state.adults;
```

Comment:
I would simplify lines 18 - 20 with `const newState = Object.assign(this.state, {});`

Reply:
@kogisu yup, great idea. Changed to reflect this suggestion